cdhweb/people/exodus.py

```diff
@@ -0,0 +1,45 @@
+"""Exodus script for people"""
+
+from mezzanine.core.models import CONTENT_STATUS_PUBLISHED
+
+from cdhweb.pages.exodus import convert_slug, get_wagtail_image, to_streamfield
+from cdhweb.people.models import OldProfile, PeopleLandingPage, Person, Profile
+
+
+def people_exodus():
+    """Exodize all existing Profiles to new Profile model, and use old Profile
+    images to update People."""
+    # all the fields on Profile that moved to Person will have been handled
+    # by django migrations; we just need to create new Page models
+    people_landing = PeopleLandingPage.objects.first()
```

Comment:
I see — define exodus methods within the app and then import and run it in the actual exodus script. Makes sense.